qa/rpc-tests/feature_zip244_blockcommitments.py

```diff
@@ -24,12 +29,10 @@ def __init__(self):
 
     def setup_nodes(self):
         return start_nodes(self.num_nodes, self.options.tmpdir, extra_args=[[
-            '-nuparams=5ba81b19:1', # Overwinter
-            '-nuparams=76b809bb:1', # Sapling
```

Comment:
Are we dropping the Overwinter and Sapling branch IDs?

Reply:
No, the test runner has been adding these by default for a while, so this just resulted in duplicate arguments being passed to the `zcashd` nodes.